Source/ThirdParty/ANGLE/src/compiler/translator/glsl/OutputGLSLBase.cpp

```diff
@@ -776,7 +776,7 @@ bool TOutputGLSLBase::visitBinary(Visit visit, TIntermBinary *node)
 
 bool TOutputGLSLBase::visitUnary(Visit visit, TIntermUnary *node)
 {
-    const char *preString  = "";
+    const char *preString; // Removed unneeded initializer to placate clang static analyzer.
```

Comment:
I'm not entirely sure this provides much value. 
If clang analyzer provides a way to skip warnings, like // NOLINT, could we use that here?